### ISSUE TITLE
automatic #showcase moderation

### ISSUE THREAD

**Issue by raylu** [strinking/futaba#359]
automatically delete #showcase posts that are just a url with no accompanying text/attachment and message the person in #bot to add a description to their showcase post

**Comment by bluedaze** [comment]
I said this in the discord channel but I thought it deserved a mention here:

Perhaps I am mistaken but... I'm actually not seeing any hard coded rules in the text filter for futaba. It looks like rules are added manually by issuing commands to the bot.

Unless I am mistaken (which is highly likely, because I suck at reading code) I don't think that adding a rule to the bot to check for messages that are only links in #showcase is something that would pass review. I think it would probably end up being something that moderators would want to set as rule by issuing a command.

**Comment by ammongit** [comment]
You're right that we want the bot to be configurable rather than hardcoded for programming specifically.

However a generic feature that can be adapted for this purpose would certainly be merged. We've been meaning to enhance the filter's capabilities and granularity, and this can be a step in that direction.

**Comment by JoshuaS3** [comment]
Does the filter cog support regular expressions? I think automatically turning every filter string into a regular expression would allow support for this and still be backwards compatible.

**Comment by ammongit** [comment]
No, there is an issue for that though
